### DISCUSSION TITLE
I can't get a subscription

### DISCUSSION THREAD

**Discussion by P4SHER** [community/community#69755] [GitHub Education]
### Hi everyone,

Your application cannot be reviewed until you fix the following:
Please correct your GitHub billing information with your last name exactly as it appears in your academic affiliation document.
Please correct your GitHub billing information with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.

I've already corrected this. Please help

**Comment by dinhducact** [comment]
Hi P4SHER,
Have you solved the problem yet?
I got the same error  as you while linking git education

**Comment by KiranZafar23** [comment]
If you've already corrected your GitHub billing information to match your academic affiliation document, but you're still encountering issues, follow these steps:

1. **Double-Check**: Ensure your first and last name in your billing information match your academic document exactly.

2. **Update Billing Info**: Edit your billing information and make the necessary changes.

3. **Re-Submit**: Once updated, re-submit your application for GitHub Education benefits.

4. **Contact GitHub**: If issues persist, reach out to GitHub support through their [contact form](https://support.github.com/contact) for further assistance.

Ensure your billing information accurately reflects your academic affiliation to facilitate the verification process.

**Comment by Dolphin-in-river** [comment]
Hi, I have this problem too. If you find a solution, please write to me :)

https://github.com/orgs/community/discussions/70008

**Comment by GT-740** [comment]
Same issue. Tried both in multiple English transcriptions and multiple native language variations. From the count of discussions it looks like it's either a recent or a regular one. Lack of GitHub support and feedback is really frustrating in this situation.

**Reply by GT-740** [reply]
Resolved by translating my first and last names with [bing translator](https://www.bing.com/translator) ([answer](https://github.com/orgs/community/discussions/68725#discussioncomment-7163466)) and using these instead of official transliterations.
